patrol/patrol.go

```diff
@@ -66,7 +69,7 @@ func (p *Patrol) GetBombSquadConfig() BombSquadMetricConfig {
 func (p *Patrol) ListSuppressedMetrics() {
 	b := p.GetBombSquadConfig()
 	for metric, labels := range b.SuppressedMetrics {
-		for label, _ := range labels {
+		for label := range labels {
```

Comment:
I think the trailing underscore is just a habit left from my Elixir experiments a few years back. :-)

Reply:
oh, this was the environment, not even me.

Just running gofmt and govet removes most of these _style_ decisions, I'm not even thinking at this level anymore.

Reply:
Oops, nevermind me. :-)